### DISCUSSION TITLE
Where is the code for System.Data.SqlClient  library?

### DISCUSSION THREAD

**Discussion by LateBeginners** [dotnet/runtime#51217] [Q&A] [answered]
Where is the code for System.Data.SqlClient  library?

**Comment by gfoidl** [comment, marked as answer]
https://github.com/dotnet/SqlClient

Cf. [Introducing the new Microsoft.Data.SqlClient](https://devblogs.microsoft.com/dotnet/introducing-the-new-microsoftdatasqlclient/)
